packages/medusa/src/api/routes/admin/discounts/__tests__/create-discount.js

```diff
@@ -284,8 +293,48 @@ describe("POST /admin/discounts", () => {
           value: 10,
           allocation: "total",
         },
+        regions: [IdMap.getId("region-france")],
         starts_at: new Date("03/14/2021 14:30"),
       })
     })
   })
+
+  describe("unsuccessful creation with invalid regions", () => {
+    let subject
+
+    beforeAll(async () => {
+      jest.clearAllMocks()
+
+      subject = await request("POST", "/admin/discounts", {
+        payload: {
+          code: "TEST",
+          rule: {
+            description: "Test",
+            type: "fixed",
+            value: 10,
+            allocation: "total",
+          },
+        },
+        adminSession: {
```

Comment:
suggestion(non-blocking): Since now there is the admin session twice, I suggest to move it at the top of the file in a const